src/variables.css

```diff
@@ -0,0 +1,29 @@
+:root {
+  --color-red: #FF3E58;
+  --color-dark-red: #59363E;
+  --color-blue: #01A4F6;
+  --color-dark-blue: #264A5E;
+  --color-dark-gray: #2F3438;
+  --color-light-gray: #727578;
+  --color-dark: #25292E;
+  --color-white: #fff;
+
+  /* TODO: delete this */
```

Comment:
TODO: delete this :)

Reply:
Done -> https://github.com/SorokinTim/covid-dashboard-rss/pull/20/commits/ed9b6daa140c6a37e0ae5eb38dde5b7876f2320f